synapse/storage/schema/delta/48/profiles_batch.sql

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2018 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ALTER TABLE profiles ADD COLUMN batch BIGINT DEFAULT NULL;
+
+CREATE INDEX profiles_batch_idx ON profiles(batch);
+
+CREATE TABLE profile_replication_status (
```

Comment:
please can we get into the habit of commenting our schema files to describe what the tables and the columns therein do.

Reply:
done